conda_build/cli/main_build.py

```diff
@@ -158,8 +158,10 @@ def parse_args(args):
 
 def output_action(metadata, config):
     silence_loggers(show_warnings_and_errors=False)
-    print(bldpkg_path(metadata, config))
-    sys.exit(0)
+    if metadata.skip():
+        print("skipped")
```

Comment:
In bioconda, we zip the output of this with the recipe names and filter accordingly. I can add more information though.